packages/react-devtools-shared/src/devtools/views/DevTools.js

```diff
@@ -127,6 +127,36 @@ export default function DevTools({
     [enabledInspectedElementContextMenu, viewAttributeSourceFunction],
   );
 
+  const devToolsRef = useRef<HTMLElement | null>(null);
+
+  useEffect(() => {
+    if (devToolsRef.current === null) return;
+
+    const ownerWindow = devToolsRef.current.ownerDocument.defaultView;
+
+    if (showTabBar) {
+      ownerWindow.addEventListener('keydown', (event: KeyboardEvent) => {
+        if (event.ctrlKey || event.metaKey) {
+          switch (event.key) {
+            case '1':
+              setTab(tabs[0].id);
+              event.preventDefault();
+              event.stopPropagation();
+              break;
+            case '2':
+              setTab(tabs[1].id);
+              event.preventDefault();
+              event.stopPropagation();
+              break;
+          }
+        }
+      });
+      return () => {
+        ownerWindow.removeEventListener('keydown');
```

Comment:
This won't actually work. `removeEventListener` needs to be passed the type *and* the handler function. You'll want to declare the handler as a separate variable, then pass it to both `addEventListener` and `removeEventListener`.

Reply:
what about?
![image](https://user-images.githubusercontent.com/36763164/76920066-3d0ef500-68d3-11ea-8e08-604ba473aed3.png)